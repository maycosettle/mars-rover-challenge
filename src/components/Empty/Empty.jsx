import { ReactComponent as FailedSearch } from 'assets/failedSearch.svg';

import useStyles from './styles';

const EmptyTable = () => {
  const styles = useStyles();

  return (
    <div className={styles.container}>
      <div className={styles.main}>
        <div className={styles.failedSearchContainer}>
          <FailedSearch />
          <h1 className={styles.noResultsTitle}>No results found</h1>
          <p className={styles.noResultsText}>Try adjusting your search or filter to find what you are looking for.</p>
        </div>
      </div>
    </div>
  );
};

export default EmptyTable;
