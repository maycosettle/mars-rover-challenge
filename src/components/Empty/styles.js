const useStyles = () => ({
  container: 'h-auto flex flex-col bg-white content-center items-center',
  main: 'w-full p-20 space-y-8 flex flex-col justify-center items-center',
  failedSearchContainer: 'flex flex-col space-y-3 bg-white content-center items-center',
  noResultsTitle: 'text-xl font-bold text-gray-200',
  noResultsText: 'max-w-sm text-center text-base font-semibold text-gray-200',
});

export default useStyles;
