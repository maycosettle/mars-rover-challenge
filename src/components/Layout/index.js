export * from './PageLayout';
export * from './PaperLayout';
export * from './PhotoLayout';
