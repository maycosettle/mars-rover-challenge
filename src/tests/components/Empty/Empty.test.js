import { shallow } from 'enzyme';
import { Empty } from 'components/Empty';

describe('tests on <Empty />', () => {
  const wrapper = shallow(<Empty />);

  test('Should render Empty correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
