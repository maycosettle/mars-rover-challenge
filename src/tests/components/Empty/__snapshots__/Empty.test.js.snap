// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <Empty /> Should render Empty correctly 1`] = `
<div
  className="h-auto flex flex-col bg-white content-center items-center"
>
  <div
    className="w-full p-20 space-y-8 flex flex-col justify-center items-center"
  >
    <div
      className="flex flex-col space-y-3 bg-white content-center items-center"
    >
      <ForwardRef(SvgFailedSearch) />
      <h1
        className="text-xl font-bold text-gray-200"
      >
        No results found
      </h1>
      <p
        className="max-w-sm text-center text-base font-semibold text-gray-200"
      >
        Try adjusting your search or filter to find what you are looking for.
      </p>
    </div>
  </div>
</div>
`;
